opal/class/opal_fifo.h

```diff
@@ -121,8 +123,12 @@ static inline opal_list_item_t *opal_fifo_pop_atomic (opal_fifo_t *fifo)
     opal_counted_pointer_t head, tail;
 
     opal_read_counted_pointer (&fifo->opal_fifo_head, &head);
-
+    int attempts = 0;
```

Comment:
This is a change in behavior; can you call it out in the commit message?

Reply:
Opps. That shouldn't be there. Thank you for catching it. The CAS does benefit from adding the release but that will be done in another PR.

Reply:
Done.